README.md

```diff
@@ -56,7 +56,7 @@ Of course, Palisade can still be deployed locally on a per-platform basis.
 
 
 ### Additional Information 
-For a quickstart in using Palisade, see the [Quickstart Guide](./QUICKSTART.md).
+For a quickstart in using Palisade, see the [Quickstart Guide](/QUICKSTART.md).
```

Comment:
This change is not required

Reply:
I think there should be one standard for links. At the moment we have three ways to write these links.

Reply:
It comes down to where they are and how they are used. This repo has special needs, since we build a gitpages and therefore the presentation is slightly different

Reply:
Ok, rolled it back